catalog.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Перфораторы в интернет-магазине «Техномарт»</title>
+  </head>
+  <body>
+<header class="header-main">
+<p class="header-logo"><a href="index.html><img src="https://via.placeholder.com/111x18" alt="Логотип «Техномарта»" width="111" height="18"></a></p>
+<!-- Здесь будет форма поиска -->
+<p class="header-bookmarks"><a href="#">Закладки: 0</a></p>
+<p class="header-cart active"><a href="#">Корзина: 10</a></p>
+<p class="header-order"><a href="#">Оформить заказ</a></p>
+<p class="header-description">Интернет-магазин<br>
+строительных материалов<br>
+и инструментов для ремонта</p>
+<div class="header-contacts">
+<span class="header-phone">+7 (812) 555-05-55</span>
+<p class="header-address">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p></div>
+<div class="user-navigation">
+<p class="user-name">Равшан Джамшутович</p>
```

Comment:
По ТЗ это ссылка на профайл
```suggestion
<a href="user-profile.html"  class="user-name">Равшан Джамшутович</a>
```